src/app/core/view/dynamic-routing/empty/empty.component.ts

```diff
@@ -0,0 +1,6 @@
+import { Component } from "@angular/core";
+
+@Component({
+  templateUrl: "empty.component.html",
+})
+export class EmptyComponent {}
```

Comment:
should rename this now, not empty anymore.
LoadingApplicationComponent or LoadingConfigComponent or BuildingAppComponent maybe?